spec/models/test_run_spec.rb

```diff
@@ -560,6 +562,38 @@
           expect(submission.current_result.overall_comment).to include(existing_comment)
         end
       end
+
+      context 'when the test run has a submission and the results contain an image annotation' do
+        let(:submission) do
+          create(:version_used_submission, grouping: grouping, submission_files: [submission_file_image])
+        end
+        let(:test_run) { create(:test_run, submission: submission) }
+
+        it 'should add the image annotation to the submission' do
+          results['test_groups'].first['annotations'] =
+            [{ 'content' => 'test annotation', 'type' => 'ImageAnnotation', 'filename' => 'test_compressed.png',
```

Comment:
Use symbol keys here: `content: 'test annotation', ...`

Same in the second test below